docs/fsharp/get-started/get-started-vscode.md

```diff
@@ -42,58 +42,50 @@ You must also install the [.NET Core SDK](https://www.microsoft.com/net/download
 
 ## Installing Visual Studio Code and the Ionide plugin
 
-You can install Visual Studio Code from the [code.visualstudio.com](https://code.visualstudio.com) website. After that, there are two ways to find the Ionide plugin:
+You can install Visual Studio Code from the [code.visualstudio.com](https://code.visualstudio.com) website.
 
-1. Use the Command Palette (Ctrl+Shift+P on Windows, ⌘+Shift+P on macOS, Ctrl+Shift+P on Linux) and type the following:
-
-    ```
-    >ext install Ionide
-    ```
-
-2. Click the Extensions icon and search for "Ionide":
-
-    ![](media/getting-started-vscode/vscode-ext.png)
+Next, click the Extensions icon and search for "Ionide":
 
```

Comment:
next line: you can also install Ionide-FAKE and to get FAKE support -> you can also install Ionide-FAKE to get FAKE support